cirq/contrib/noise_models/noise_models.py

```diff
@@ -211,54 +212,142 @@ def noisy_moment(self, moment: 'cirq.Moment',
             ]
 
 
-class PerQubitDepolarizingNoiseModel(devices.NoiseModel):
-    """DepolarizingNoiseModel which allows depolarization probabilities to be
-    specified separately for each qubit.
+class PerQubitDepolarizingWithDampedReadoutNoiseModel(devices.NoiseModel):
+    """DepolarizingWithDampedReadoutNoiseModel which allows depolarization
+    probabilities to be specified separately for each qubit.
 
-    Similar to depol_prob in DepolarizingNoiseModel, depol_prob_map should map
-    Qids in the device to their depolarization probability.
+    This simulates asymmetric readout error. The noise is structured
+    so the T1 decay is applied, then the readout bitflip, then measurement.
+    Note that T1 decay is only applied to measurement, not other gates.
+
+    In moments without measurement, all qubits affected by an operation will
+    have a depolarizing channel applied after the original operation. Qubits
+    that remain idle will be unaffected by this model.
+
+    As with the DepolarizingWithDampedReadoutNoiseModel, if a circuit contains
+    measurements, they must be in moments that don't also contain gates.
     """
 
     def __init__(
             self,
-            depol_prob_map: Dict['cirq.Qid', float],
+            depol_prob_map: Dict['cirq.Qid', float] = None,
```

Comment:
nit: would call this `depol_probs` since type here really tells you that it is a map (also python people would say `_dict`).  similar for other variables

Reply:
Done (opted for `_probs` suffix in all cases)